packages/clerk-js/bundlewatch.config.json

```diff
@@ -2,8 +2,8 @@
   "files": [
     { "path": "./dist/clerk.js", "maxSize": "560kB" },
     { "path": "./dist/clerk.browser.js", "maxSize": "75kB" },
-    { "path": "./dist/clerk.headless.js", "maxSize": "48.3KB" },
-    { "path": "./dist/ui-common*.js", "maxSize": "89KB" },
+    { "path": "./dist/clerk.headless.js", "maxSize": "48.6KB" },
```

Comment:
Bundle increased a bit because of all the `Clerk` core changes. Fortunally tho we could apply lazy loading and avoid increasing the bundle size for sign-in/sign-up